### PR TITLE
Fix types path and add "@types/react-native" as dev dependencies.

### PR DESCRIPTION
**Pull request by AndreOneti**
I have a problem with types path on react-native project with typescript.

I changed types on package.json and works for me, and testing with source code, it's work fine.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
     "react-native": "*"
   },
   "devDependencies": {
+    "@types/react-native": "^0.71.4",
```
Comment (bastengao) [inline package.json:46]: I think it's better same with react-native.

```suggestion
    "@types/react-native": "^0.59.10",
```

Comment (AndreOneti) [reply package.json:46]: I checked the versions of "@types/react-native" but it doesn't have the version "0.59.10".

image

The suggestion is to update react-native to version "0.71.4" as well.

Comment (bastengao) [reply package.json:46]: OK. It's better to choose "[0.60.31](https://www.npmjs.com/package/@types/react-native/v/0.60.31)"

Comment (AndreOneti) [reply package.json]: Version updated.

**Review by bastengao — APPROVED** [review]

**Pull request merged**